app/assets/javascripts/organisation-autocomplete.js

```diff
@@ -5,16 +5,118 @@ window.GOVUKAdmin.Modules = window.GOVUKAdmin.Modules || {};
 
   Modules.OrganisationAutocomplete = function () {
     this.start = function ($element) {
-      accessibleAutocomplete.enhanceSelectElement({
-        selectElement: $element.find('#organisations')[0],
-        showAllValues: true,
-        autoselect: false,
-        dropdownArrow: dropdownArrow
-      });
-    };
 
-    function dropdownArrow() {
-      return '<span class="caret"></span>';
-    }
+      var autocompleteTemplateHtml = createAutocompleteTemplate();
+
+      $element
+        .append(addOrganisationButton())
+        .find('[selected]')
+        .each(function (index, option) {
+          if (index > 0) {
+            $(autocompleteTemplateHtml)
+              .insertAfter(lastSelectWrapper())
+              .find('option[value="' + $(option).attr('value') + '"]')
```

Comment:
There's got to be a nicer way of doing this. Probably using [`.val()`](https://api.jquery.com/val/).

Reply:
👍